src/RetryOnAuthorizationError.php

```diff
@@ -34,6 +34,6 @@ public function __invoke(
 
     private function isUnauthorizedResponse(int $retries, ResponseInterface $response = null)
     {
-        return !empty($response) && $retries < 1 && $response->getStatusCode() === 401;
+        return !empty($response) && $retries < 1 && in_array($response->getStatusCode(), [401,403]);
```

Comment:
Should we retry on 403? That should mean that the token is valid, but you do not have the right permissions to perform the action.

Reply:
In my case the token had just expired (I forced the server to expire it in order to test). Instead of returning 401 it returned 403. Not what I was expecting but a hard to change behavior. Since is just one retry I believe it is worth.

Reply:
I think an expired token should return 401(Unauthorized) and not 403 (Forbidden), that's most likely a server issue.

Reply:
Well, it won't be a easy fit to change that for a Drupal serving graphql. 
You're probably right since RFC specifically state client shouldn't retry after receiving 403 errors.

Reply:
In this case I would suggest injecting in the config an array of status codes on which the library should perform the retry: this would fix the issue of not RFC compliant backends (as in your example, 403) and flaky backends (500, 502).